jlm/llvm/ir/operators/lambda.hpp

```diff
@@ -380,7 +402,7 @@ class node final : public jlm::rvsdg::structural_node
    *
    * \return The output of the lambda node.
    */
-  lambda::output *
+  rvsdg::output *
```

Comment:
There are 179 call sites, is this really worth it doing now? I would rather figure a way to restructure the construction API (the "builder" pattern as used for phi actually looks very suitable) instead of causing more churn.

Reply:
Okay.